src/registrar/tests/test_models.py

```diff
@@ -1868,10 +1899,11 @@ def test_is_tribal_complete(self):
         self.domain_request.is_election_board = False
         self.domain_request.save()
         self.assertTrue(self.domain_request._is_tribal_complete())
-        self.domain_request.tribe_name = None
         self.domain_request.is_election_board = None
         self.domain_request.save()
-        # is_election_board will overwrite to False bc of _update_org_type_from_generic_org_and_election
+        self.assertFalse(self.domain_request._is_tribal_complete())
+        self.domain_request.tribe_name = None
+        self.domain_request.save()
         self.assertFalse(self.domain_request._is_tribal_complete())
```

Comment:
Is_tribal_complete will fail if either tribe_name or is_election_board are None, but I didn't test the case you described because it's not possible with the normal request flow.